dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessInstanceController.java

```diff
@@ -204,6 +205,29 @@ public Result queryProcessInstanceById(@ApiIgnore @RequestAttribute(value = Cons
         return returnDataList(result);
     }
 
+    @ApiOperation(value = "queryTopNLongestRunningProcessInstance", notes = "QUERY_TOPN_LONGEST_RUNNING_PROCESS_INSTANCE_NOTES")
```

Comment:
Hi,
It will be better to add the java docs like[1] here. can you do it?
[1] ![image](https://user-images.githubusercontent.com/29545877/88796597-8140c900-d1d4-11ea-87a6-3d115faafa69.png)

Reply:
ok , I will do it in next commit